### PR TITLE
checkbox fix for #575

### PR DESCRIPTION
**Pull request by sendilkumarn**
#575

### CODE REVIEW

**Comment by jelbourn** [comment]
@sendilkumarn Can you title your PRs such that it's clear what they're doing, and include some description that has a high level summary of your changes and why they fix the issue? Seeing just "checkbox fix" makes it more difficult to review.

File: src/components/checkbox/checkbox.html [hunk 1]
```diff
@@ -11,7 +11,8 @@
            [attr.aria-labelledby]="ariaLabelledby"
            (focus)="onInputFocus()"
            (blur)="onInputBlur()"
-           (change)="onInteractionEvent($event)">
+           (change)="onInteractionEvent($event)"
+           (click)="onInteractionEvent($event)">
```
Comment (jelbourn) [inline src/components/checkbox/checkbox.html]: This doesn't seem right. What is the underlying bug you're trying to address?

**Comment by jelbourn** [comment]
This change will also need a unit test.

File: src/components/checkbox/checkbox.html (hunk 1 above)
Comment (sendilkumarn) [inline src/components/checkbox/checkbox.html]: upon clicking, checkbox fires click function twice. Ideally it should fire it once right?

**Comment by sendilkumarn** [comment]
For PR - I will :)

Unit Test, I totally forgot. I will add it by today.

**Comment by sendilkumarn** [comment]
I will close this PR and add another.

**Pull request closed without merging**